### PR TITLE
write rd

### PR DESCRIPTION
**Pull request by sgharms**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,16 +1,120 @@
...
+What if we were to make an expression that had no _variables_, since we haven't
+learned about them yet (but will soon!). What if it _also_ didn't have
+_operators_? That would be an expression like:
+
+```ruby
+42
+```
+
+This expression is the _constant expression_ and it's very important, although
+very boring From a programming as conversation perspective, the _constant
```
Comment (maxwellbenton) [inline README.md]: from

File: README.md [hunk 2]
```diff
@@ -1,16 +1,120 @@
...
+Let's consider a simple arithmetic expression. Keep in mind we apply operators
+in "[PEMDAS][]" order: parenthesis, exponents, multiplication, division,
+addition, subtraction.
+
+| Expression | Has Operators? | Operators | Are we done?| Which to Apply |
+| ---------- | -------------- | --------- | ------------| -------------- |
+| 3 * ( 10 - 5 ) | YES | `*`, `()`| NO | Open new sub-expression in `()` because of PEMDAS|
+| ( 10 - 5 )| YES | `-`| NO | `-` because of PEMDAS|
+| 5 | NO | NONE | YES| Constant expression! Return the value of the constant, we're done!|
+| 3 * 5 | YES | NO | Swap in the return value of the inner expression|
```
Comment (maxwellbenton) [inline README.md]: doesn't this have operators (*)?

File: README.md [hunk 3]
```diff
@@ -1,16 +1,120 @@
...
+"Mirroring" establishes "I see what you see" and "I can 'follow' your
+analogies, we can build on this."
+
+Earth. Shaking. Power.
+
+From simple "mirroring" you can build to trading "what we call something"
+versus "what you call something."
+
+![Building](http://gph.is/2eY5zh1)
+
```
Comment (maxwellbenton) [inline README.md:96]: is this meant to be the same GIF as the previous one?

File: README.md [hunk 4]
```diff
@@ -1,16 +1,120 @@
...
+Because of _agreement_ exists, we can _trust_ our conversational partner.  What
+if when we told Ruby `1 + 1`, it returned `3`. Or what if _sometimes_ it
```
Comment (maxwellbenton) [inline README.md]: missing question mark

File: README.md (hunk 2 above)
Comment (sgharms) [reply README.md]: Ughg, hard to edit on a laptop. improved.

**Pull request merged**